Client/Frontend/Browser/TabManager.swift

```diff
@@ -16,7 +16,7 @@ class TabManager {
     var delegate: TabManagerDelegate? = nil
 
     private var tabs: [Browser] = []
-    private var selectedIndex = -1
+    var selectedIndex = -1
```

Comment:
Please make this a getter so that external classes can't modify it. Bad things would happen if they do.